### PR TITLE
feat: add base sepolia

### PR DESCRIPTION
**Pull request by dtbuchholz**
## Summary

- Adds Base Sepolia support via SDK bump
- Also, updates ethers to 6.12.1 to align with other tableland packages

### CODE REVIEW

**Comment by dtbuchholz** [comment]
@joewagner dang, i forgot about the circular CLI dependency w/ this and the normal CLI. so, we'll need to release the Studio CLI, and then we can update the normal `tableland-js` CLI as a patch version?

(also, i _defintely_ ran `npm install` in root for this one!)

**Comment by joewagner** [comment]
This looks good!  Let's wait until Monday to release this. 
Also, there is a fix for the failing nonce tests in #273, which it looks like might be causing a problem here.

File: packages/chains/src/index.ts [hunk 1]
```diff
@@ -57,6 +58,7 @@ function supportedChains(isLocalDev = false) {
     { ...arbitrum, testnet: false },
     { ...arbitrumSepolia, testnet: true },
     { ...arbitrumNova, testnet: false },
+    { ...baseSepolia, testnet: false },
```
Comment (asutula) [inline packages/chains/src/index.ts:61]: It is a testnet, right?

Comment (dtbuchholz) [reply packages/chains/src/index.ts:61]: Oh shoot, no. Copied the line above and didn't fix that. Will change it.

**Comment by dtbuchholz** [comment]
ok, fixed that mislabeled `mainnet` -> `testnet`

**Review by asutula — APPROVED** [review]

**Pull request merged**